docs/user/openstack/connect_nodes_to_ipv6_network.md

```diff
@@ -0,0 +1,59 @@
+# Connect workers nodes and Pods to IPv6 network
+
+To connect your workers at the time of installation you can use [additionalNetworkIDs](https://github.com/openshift/installer/blob/master/docs/user/openstack/customization.md#additional-networks) parameter in the install config and set IPv6 network ID there:
+
+Example OpenShift install config:
+
+```yaml
+...
+compute:
+- name: worker
+  platform:
+    openstack:
+      additionalNetworkIDs:
+      - <ipv6_network_id>
+...
+```
+
+**Note**: To use Stateful IPv6 Networks, the arg `ip=dhcp,dhcpv6` needs to be included in the kernal args of the Worker nodes, otherwise the Nodes won't get an IPv6 address. Use the [procedure][add-kernel-args] to add kernel argument to the Nodes.
+
+## Enable connectivity to the Pods
```

Comment:
We should explain for what reason we recommend to disable the port security. That's because pods get random MAC addresses and this would force us to constantly change the allowed address pairs, after a new pod that is connected to this network is created. That also means that pods wouldn't be able to communicate on the network straight away, at least not until the port knew about the new allowed address pair.

Reply:
Good point. I just added it.